tests/test_styler.py

```diff
@@ -141,7 +142,8 @@ def test_highlight_quantile() -> None:
 
 
 def test_loader() -> None:
-    check(assert_type(DF.style.loader, PackageLoader), PackageLoader)
+    if PD_LTE_23:
+        check(assert_type(DF.style.loader, PackageLoader), PackageLoader)
```

Comment:
This is for the nightly, in 3.0.0 it will return a `FileSystemLoader` (see https://github.com/pandas-dev/pandas/pull/62123)

Reply:
Can you add a comment here?